components/api/db/queries.bal

```diff
@@ -95,3 +95,28 @@ public const string SEARCH_ORGS_QUERY_IMAGE_COUNT = "SELECT REGISTRY_ORGANIZATIO
                                     FROM REGISTRY_ORGANIZATION LEFT JOIN
                                     REGISTRY_ARTIFACT_IMAGE ON REGISTRY_ORGANIZATION.ORG_NAME=REGISTRY_ARTIFACT_IMAGE.ORG_NAME
                                     WHERE REGISTRY_ORGANIZATION.ORG_NAME LIKE ? GROUP BY REGISTRY_ORGANIZATION.ORG_NAME LIMIT ? OFFSET ?";
+                                
+public const string SEARCH_USER_ORGS_TOTAL_COUNT = "SELECT COUNT(ORG_NAME) FROM REGISTRY_ORG_USER_MAPPING
+                                    WHERE ORG_NAME LIKE ? AND USER_UUID = ?";
+
+public const string SEARCH_USER_ORGS_QUERY = "SELECT REGISTRY_ORGANIZATION.ORG_NAME, REGISTRY_ORGANIZATION.DESCRIPTION,
+                                    COUNT(REGISTRY_ORG_USER_MAPPING.USER_UUID) AS MEMBER_COUNT
+                                    FROM REGISTRY_ORGANIZATION LEFT JOIN
+                                    REGISTRY_ORG_USER_MAPPING ON REGISTRY_ORGANIZATION.ORG_NAME=REGISTRY_ORG_USER_MAPPING.ORG_NAME
+                                    WHERE REGISTRY_ORGANIZATION.ORG_NAME LIKE ? AND REGISTRY_ORG_USER_MAPPING.USER_UUID = ?
+                                    GROUP BY REGISTRY_ORGANIZATION.ORG_NAME LIMIT ? OFFSET ?";
+
+public const string SEARCH_USER_ORGS_QUERY_IMAGE_COUNT = "SELECT REGISTRY_ORGANIZATION.ORG_NAME, COUNT(REGISTRY_ARTIFACT_IMAGE.ARTIFACT_IMAGE_ID) as Image_Count
+                                    FROM REGISTRY_ORGANIZATION LEFT JOIN
+                                    REGISTRY_ARTIFACT_IMAGE ON REGISTRY_ORGANIZATION.ORG_NAME=REGISTRY_ARTIFACT_IMAGE.ORG_NAME 
+                                    WHERE (REGISTRY_ARTIFACT_IMAGE.VISIBILITY = \"PUBLIC\" OR REGISTRY_ARTIFACT_IMAGE.VISIBILITY is NULL) AND
```

Comment:
VISIBILITY is NOT NULL right ?

Reply:
Yes. Removing REGISTRY_ARTIFACT_IMAGE.VISIBILITY is NULL

Reply:
Yes. It is NOT NULL.
But this check is needed to find organizations with 0 images.